generators/quarkus/entity-files.js

```diff
@@ -68,7 +68,12 @@ export const entityServerFilesFromJHipster = {
             condition: generator =>
                 generator.dtoMapstruct &&
                 (generator.databaseType === 'sql' || generator.databaseType === 'mongodb' || generator.databaseType === 'couchbase'),
-            templates: ['_entityPackage_/service/mapper/_entityClass_MapperTest.java'],
+            templates: [
+                '_entityPackage_/domain/_persistClass_Asserts.java',
+                '_entityPackage_/domain/_persistClass_Test.java',
+                '_entityPackage_/domain/_persistClass_TestSamples.java',
+                '_entityPackage_/service/mapper/_entityClass_MapperTest.java',
```

Comment:
@mshima These files are not found when running the generator. Do I have to do something to pull in the templates from the `java` generator? 

> FATAL! Error: Template file src/test/java/_package_/_entityPackage_/domain/_persistClass_Asserts.java was not found at /Users/mraible/dev/generator-jhipster-quarkus/generators/server/templates,/Users/mraible/dev/generator-jhipster-quarkus/node_modules/generator-jhipster/dist/generators/server/templates

Reply:
You should set entityDomainLayer to true in entity at preparingEachEntity.
This will make the Java generator to generate domain files.

Reply:
Or add Java generator like this:
https://github.com/jhipster/generator-jhipster-micronaut/blob/1a39ef5e54608875d5b1ae068f4afdc737517273/generators/server/generator.js#L29

Reply:
Adding the templates from the Java generator helps. However, I can't seem to override `_entityPackage_/service/mapper/_entityClass_MapperTest.java` with a local template. Added in https://github.com/jhipster/generator-jhipster-quarkus/pull/341/commits/d4b9c1115cad4facf39e39e6364279580f2ea47d.

Reply:
Nevermind, I figured it out.